src/window.py

```diff
@@ -189,6 +191,7 @@ def _setup(self):
         places.connect("failed", self._on_places_failed)
         places.connect("unlock", self._on_places_unlock)
         self.places_inner_box.append(places)
+        self._places = places
```

Comment:
Once we get a proper PortfolioBookmarks model, there would be no need to add this widget dependency here and, instead, the window widget would only need t interact with the PorfolioBookmarks model.

In other words, both PortfolioPlaces and PortfolioWindow would interact with the PortfolioBookmarks model.